imap_processing/tests/codice/test_codice_l1a.py

```diff
@@ -41,7 +43,6 @@ def test_codice_l1a(l0_test_data: list[space_packet_parser.parser.Packet]) -> st
         pytest fixture used to provide a temporary directory during testing
     """
 
-    cdf_filename = codice_l1a(l0_test_data)
+    cdf_filename = process_codice_l1a(l0_test_data, tmp_path)
```

Comment:
You shouldn't need the `tmp_path` now after the dev changes.

```suggestion
    cdf_filename = process_codice_l1a(l0_test_data)
```